### PR TITLE
Make @alexwolfmsft a codeowner of Azure stuff

### PR DESCRIPTION
**Pull request by CamSoper**
So he doesn't need @jsquire approval on everything. Also making it so I can approve PRs for him

cc: @alexwolfmsft

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -61,21 +61,21 @@
 /docs/azure/includes/ @dotnet/docs
 
 # Azure SDK catch-all entry
-/docs/azure/sdk/ @jsquire @pallavit
+/docs/azure/sdk/ @jsquire @pallavit @alexwolfmsft
```
Comment (jsquire) [inline .github/CODEOWNERS]:
```suggestion
/docs/azure/sdk/ @jsquire @alexwolfmsft @scottaddie
```

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -61,21 +61,21 @@
...
-/docs/azure/sdk/azure-sdk-for-net.md @jsquire @pallavit
-/docs/azure/sdk/dependency-injection.md @JoshLove-msft @jsquire
-/docs/azure/sdk/logging.md @annelo-msft @christothes @JoshLove-msft @KrzysztofCwalina @jsquire
-/docs/azure/sdk/packages.md @jsquire @pallavit
-/docs/azure/sdk/pagination.md @annelo-msft @christothes @JoshLove-msft @KrzysztofCwalina @jsquire
-/docs/azure/sdk/resource-management.md @ArthurMa1978 @m-nash
-/docs/azure/sdk/thread-safety.md @annelo-msft @christothes @JoshLove-msft @KrzysztofCwalina @jsquire
-/docs/azure/sdk/unit-testing-mocking.md @annelo-msft @christothes @JoshLove-msft @KrzysztofCwalina @jsquire
+/docs/azure/sdk/azure-sdk-configure-proxy.md @annelo-msft @christothes @JoshLove-msft @KrzysztofCwalina @jsquire @alexwolfmsft
+/docs/azure/sdk/azure-sdk-for-net.md @jsquire @pallavit @alexwolfmsft
```
Comment (jsquire) [inline .github/CODEOWNERS]:
```suggestion
/docs/azure/sdk/azure-sdk-for-net.md @jsquire @scottaddie @alexwolfmsft
```

File: .github/CODEOWNERS [hunk 3]
```diff
@@ -61,21 +61,21 @@
...
+/docs/azure/sdk/dependency-injection.md @JoshLove-msft @jsquire @alexwolfmsft
+/docs/azure/sdk/logging.md @annelo-msft @christothes @JoshLove-msft @KrzysztofCwalina @jsquire @alexwolfmsft
+/docs/azure/sdk/packages.md @jsquire @pallavit @alexwolfmsft
```
Comment (jsquire) [inline .github/CODEOWNERS]:
```suggestion
/docs/azure/sdk/packages.md @jsquire @scottaddie @alexwolfmsft
```

**Review by jsquire — COMMENTED** [review]
Since Pallavi is no longer with us, let's throw Scott in there as well, please.

**Review by alexwolfmsft — APPROVED** [review]

**Review by IEvangelist — APPROVED** [review]

**Pull request merged**